builtin/files/userdata/cloud-config-controller

```diff
@@ -1124,9 +1131,10 @@ write_files:
       remove "${mfdir}/canal.yaml"
       remove "${mfdir}/calico.yaml"
       {{- else if eq .Kubernetes.Networking.SelfHosting.Type "calico" }}
+      # TODO disabled the following removals since it is messing with already existent resources that are shared between the various configs.
```

Comment:
This concerns me a little bit. If somebody deploys canal and then tries to deploy Calico isn't this going to cause there to be a mess of resources?

Reply:
Yeah, very problematic. Cool if we open an issue to get that figured out outside this PR?